### PR TITLE
#Trivial Correct f-mega-modal README formatting.

### PR DESCRIPTION
**Pull request by BanksySan**
Correct the heading levels as they jumped from level 1 to level 3.

### CODE REVIEW

**Review by DamianMullins — DISMISSED** [review]

**Review by kevinrodrigues — DISMISSED** [review]

File: packages/components/molecules/f-mega-modal/README.md [hunk 1]
```diff
@@ -17,7 +16,7 @@
 
 ## Usage
 
-### Installation
+## Installation
```
Comment (ashleynolan) [inline packages/components/molecules/f-mega-modal/README.md:19]: This looks correct to me as-is. Line 17 is `h2`, and these are then `h3`?

**Review by xander-marjoram — CHANGES_REQUESTED** [review]
I agree with Ash, I think it was correct already.

**Review by ashleynolan — DISMISSED** [review]
Just request comment before it can be merged

File: packages/components/molecules/f-mega-modal/README.md (hunk 1 above)
Comment (BanksySan) [reply packages/components/molecules/f-mega-modal/README.md:19]: Yup.  The **Usage** being a level two confused me.  I've made it level 1.

**Review by xander-marjoram — APPROVED** [review]

**Review by kevinrodrigues — APPROVED** [review]

File: packages/components/molecules/f-mega-modal/README.md (hunk 1 above)
Comment (ashleynolan) [reply packages/components/molecules/f-mega-modal/README.md:19]: There's already an `h1` up on line 3, so I think it was still correct previously

**Pull request merged**